nova/tests/unit/compute/test_utils.py

```diff
@@ -1446,6 +1446,80 @@ def test_upsize_quota_delta_quota_separate_both_different(self):
         deltas = compute_utils.upsize_quota_delta(new_flavor, old_flavor)
         self.assertEqual(expected_deltas, deltas)
 
+    def test_upsize_quota_delta_towards_hw_version(self):
+        """Resizing from not having a hw_version towards a hw_version
+        needs all resources in addittion
```

Comment:
```suggestion
        needs all resources in addition
```
and everywhere below too.